monitoring/jaeger-mixin/README.md

```diff
@@ -0,0 +1,80 @@
+# Prometheus monitoring mixin for Jaeger
+
+The Prometheus monitoring mixin for Jaeger provides a starting point for people wanting to monitor Jaeger using Prometheus, Alertmanager, and Grafana. To use it, you'll need [`jsonnet`](https://github.com/google/go-jsonnet) and [`jb` (jsonnet-bundler)](https://github.com/jsonnet-bundler/jsonnet-bundler). They can be installed using `go get`, as follows:
+
+```console
+$ go get github.com/google/go-jsonnet/cmd/jsonnet
+$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+```
+
+Your monitoring mixin can then be initialized as follows:
+
+```console
+$ jb init
+$ jb install \
+  github.com/jaegertracing/jaeger/monitoring/jaeger-mixin@master \
+  github.com/grafana/jsonnet-libs/grafana-builder@master \
+  github.com/coreos/kube-prometheus/jsonnet/kube-prometheus@master
+```
+
+Your mixin can then look like this:
```

Comment:
I haven't run the command - so may be obvious if I did - but where is this mixin generated, such that it may look like the following? Does the user have to do anything specific with the mixin? If not, then is there a benefit in including it in this doc?

Reply:
I'll improve this part, but our mixins are intended to be consumed as a library. So, users would declare a dependency on our mixin and specify their own jsonnet file making use of it.

Reply:
Ah ok - difficult to know how much explanation should be added - depends on user's knowledge of jsonnet.

Reply:
I changed it to read:

> In the directory your mixin was initialized, create a new `monitoring-setup.jsonnet` file like the following. Note that this file is yours, any customizations to Prometheus, Grafana, or Alertmanager should take place here.